vita3k/config/include/config/config.h

```diff
@@ -17,47 +17,49 @@
 
 #pragma once
 
-#include <util/fs.h>
+#include <psp2/system_param.h>
+#include <spdlog/spdlog.h>
 
-#include <boost/optional.hpp>
+// clang-format off
+// Singular options produced in config file
+// Order is code(option_type, option_name, default_value)
+// When adding in a new macro for generation, ALL options must be stated.
+#define CONFIG_INDIVIDUAL(code)                                                                 \
+    code(bool, log_imports, false)                                                              \
```

Comment:
Put the name of config in "". I dobt like the underscore, and i doubt "" will bite

Reply:
You mean like this?

`code(bool, "log_imports", false)`

Reply:
Yeah, also i like to keep the name like before ( log-imports )

Reply:
Those requests will actually be a problem.

e.g., note that the macro `code(bool, "log-imports", false) becomes `YAML_MEMBER(bool, "log-imports", false)` which becomes `bool "log-imports" = false`, which doesn't make sense in C++ and fails equally elsewhere in the PR. Additionally, dashes in names (without quotes) are not valid variables, and it is too difficult/unknown at least from my searching to find out how to replace "-" with "_" (and it is better to make the names uniform). An `std::map<std::string, std::variant>` would do the trick (and is the easier option to get to what you want, and we are in C++17) and is not too hard to implement (relatively speaking), but doing it will be pretty drastic for Vita3K, since it is currently getting the variables as members and not as a map/dictionary *all over the place*.

So, it is either getting the members from a dictionary-like object, or keep them as they are and use the member variables instead. If you want to do the dictionary route, I wouldn't mind; it will take some time, though.